content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -334,6 +334,15 @@ are not vulnerable to ordering changes in the list.
 Once the last finalizer is removed, the resource is actually removed from etcd.
 
 
+## Single resource API
+
+All API endpoints support single resources only. There is currently no API support
```

Comment:
```suggestion
All API endpoints support single resources only. There is no API support
```
based on https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future

Reply:
list/watch support reading multiple resources, and deletecollection supports deleting multiple resources in a single request